Dockerfile

```diff
@@ -33,7 +33,8 @@ RUN zypper install -y \
     libguestfs kernel-default e2fsprogs parted gptfdisk btrfsprogs \
     podman \
     createrepo_c \
-    nm-configurator
+    nm-configurator \
+    guestfs-tools lvm2
```

Comment:
Nit: We can move this to the second line (`libguestfs...`) since those are the dependencies we've specified we need for RAW image modification.

Reply:
Done!